host/esp_modem_usb_dte/idf_component.yml

```diff
@@ -1,7 +1,7 @@
 ## IDF Component Manager Manifest File
 version: "1.1.0~1"
 description: USB DTE plugin for esp_modem component
-url: https://github.com/espressif/idf-extra-components/tree/master/usb/esp_modem_usb_dte
+url: https://github.com/espressif/esp-bsp/tree/master/host/esp_modem_usb_dte
```

Comment:
@igrr @tore-espressif  BTW, Does GitHub have a method to redirect the previous URLs from `esp-exra-components` ? Because some of them we've referenced elsewhere, or recommended to our customers

Reply:
@leeebo In case of [ESP-Registry](https://components.espressif.com/) we can retrospectively modify the URLs for already released versions.

IIRC, GitHub won't redirect for security reasons cc @kumekay @tomassebestik